api/config/config_preprod.json

```diff
@@ -55,6 +55,7 @@
     "extractionCohortsDataset": "fc-aou-cdr-preprod-ct.wgs_extraction_cohorts",
     "extractionDestinationDataset": "fc-aou-cdr-preprod-ct.wgs_extraction_destination",
     "extractionTempTablesDataset": "fc-aou-cdr-preprod-ct.wgs_extraction_temp_tables",
+    "gatkJarUri": "gs:\/\/all-of-us-workbench-test-genomics\/wgs\/gatk-package-4.2.0.0-131-g2c1fc16-SNAPSHOT-local.jar",
```

Comment:
FYI: this dependency is not ideal, but it is the existing behavior. https://precisionmedicineinitiative.atlassian.net/browse/RW-7261 tracks a fix there (I need to create a new bucket and configure permissions to resolve this - which is why I'm not changing it here)

Reply:
I don't have enough context, but another direction that we might be able to explore is to use the jar hosted in Broad jfrog. Add a gradle task that download jar from jfrog, and put is under `resource/`

Example: (it did a step further, i..e unzip that jar file)
https://github.com/DataBiosphere/terra-resource-buffer/blob/master/build.gradle#L193L204

Reply:
Thanks. The GATK jar is needed in a Cromwell workflow - not on the spring server, though pushing the jar to a package repo makes sense. Trying to figure this out with the Broad team.

Reply:
Thanks. I did see those jars in Broad's jfrog, e.g. : https://broadinstitute.jfrog.io/artifactory/libs-snapshot-local/org/broadinstitute/gatk/4.2.0.0-36-gc84eaee-SNAPSHOT/gatk-4.2.0.0-36-gc84eaee-20210609.142057-1.jar